product-features/farming/how-to.md

```diff
@@ -0,0 +1,208 @@
+---
+description: >-
+  Step-by-step guides to learn how to stake LP tokens into a farm, harvest
+  rewards and remove LP tokens from the farm.
+---
+
+# How to farm & harvest
+
+Yield farming takes a few easy steps to get set up. It works very similar to standard staking, with the key difference being that the tokens you stake are LP tokens. As in traditional staking, you lock up your tokens for a certain period (measured in cycles) and earn rewards over time. After each (except for the first cooldown one), you will have rewards available to harvest.
```

Comment:
"As in traditional staking" -> "As with traditional staking"

Reply:
Corrected